app/models/battle.rb

```diff
@@ -104,6 +106,8 @@ class Entity < Grape::Entity
     expose :state
     expose :ruleset_id, as: :rulesetId
     expose :winner
+    expose :diff
+    expose :balance
```

Comment:
I want to set the balance "team" assignment on the `BattlePlayer` record. That way the front end can use battle players however it wants (ie, arrays, maps, etc) rather than just as one string.